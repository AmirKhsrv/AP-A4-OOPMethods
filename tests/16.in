best_book
show_author_info 1678
credit 116
show_sorted_shelf 55 currently_reading Fantasy
best_book
recommend_second_approach 7
best_book
show_author_info 1030
show_author_info 24
credit 54
show_sorted_shelf 38 want_to_read HolidayHorror
recommend_first_approach 125
show_author_info 1594
show_author_info 1278
best_book
