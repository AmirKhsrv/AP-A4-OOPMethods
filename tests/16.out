id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 1678
Name: Kevin Brockmeier
Year of Birth: 1956
Place of Birth: Holland
Member Since: March 10 2016
Genres: Children S Books, Fiction, Science Fiction
Books:
0.013154
id: 199
Title: Valour and Vanity
Genre: Fantasy
Author: Mary Robinette Kowal
***
id: 395
Title: Dirty Ross
Genre: Romance
Author: Sandrine Gasq-Dion
***
id: 174
Title: Five Days at Memorial: Life and Death in a Storm-Ravaged Hospital
Genre: Nonfiction
Author: Sheri  Fink
***
id: 113
Title: Ghost in the Shell
Genre: Sequential Art
Author: Masamune Shirow
***
id: 144
Title: Lies My Teacher Told Me: Everything Your American History Textbook Got Wrong
Genre: History
Author: James W. Loewen
***
id: 186
Title: PODs
Genre: Science Fiction
Author: Michelle K. Pickett
***
id: 279
Title: Raphael
Genre: Paranormal
Author: D.B. Reynolds
***
id: 92
Title: Rock Crystal
Genre: Fiction
Author: Adalbert Stifter
***
id: 172
Title: The Last Kingdom
Genre: Historical
Author: Bernard Cornwell
***
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
Reviews:
id: 99 Rating: 3 Likes: 8 Date: November 16 2018
id: 142 Rating: 4 Likes: 0 Date: January 20 2019
id: 949 Rating: 1 Likes: 1 Date: May 14 2017
id: 1209 Rating: 1 Likes: 0 Date: August 22 2013
id: 1350 Rating: 5 Likes: 2 Date: September 25 2018
id: 1595 Rating: 1 Likes: 0 Date: July 1 2013
id: 1747 Rating: 1 Likes: 0 Date: November 13 2018
id: 1846 Rating: 4 Likes: 0 Date: July 9 2016
id: 3401 Rating: 3 Likes: 0 Date: January 1 2016
id: 3889 Rating: 5 Likes: 0 Date: October 17 2017
id: 4887 Rating: 5 Likes: 3 Date: May 4 2018
id: 5127 Rating: 5 Likes: 0 Date: November 9 2010
id: 5574 Rating: 3 Likes: 0 Date: October 6 2014
id: 6137 Rating: 3 Likes: 0 Date: June 28 2017
id: 6480 Rating: 1 Likes: 1 Date: May 1 2011
id: 7559 Rating: 0 Likes: 7 Date: August 19 2017
id: 7913 Rating: 3 Likes: 0 Date: July 10 2019
id: 8048 Rating: 2 Likes: 0 Date: December 23 2018
id: 8160 Rating: 1 Likes: 0 Date: February 26 2017
id: 8310 Rating: 2 Likes: 0 Date: November 29 2019
id: 8626 Rating: 5 Likes: 0 Date: June 16 2014
id: 8683 Rating: 4 Likes: 0 Date: June 20 2013
id: 8858 Rating: 0 Likes: 0 Date: December 16 2015
id: 8862 Rating: 4 Likes: 0 Date: August 26 2018
id: 8903 Rating: 5 Likes: 0 Date: July 11 2018
id: 9182 Rating: 2 Likes: 0 Date: October 12 2018
id: 10427 Rating: 2 Likes: 0 Date: July 28 2018
id: 10432 Rating: 3 Likes: 0 Date: January 9 2018
id: 10490 Rating: 4 Likes: 0 Date: October 20 2017
id: 11047 Rating: 2 Likes: 24 Date: October 15 2010
id: 12102 Rating: 2 Likes: 3 Date: April 22 2018
id: 12147 Rating: 4 Likes: 2 Date: December 12 2019
id: 12633 Rating: 3 Likes: 0 Date: March 21 2016
id: 13302 Rating: 4 Likes: 0 Date: April 18 2015
id: 14111 Rating: 3 Likes: 1 Date: September 14 2016
id: 14698 Rating: 2 Likes: 0 Date: October 22 2019
id: 16634 Rating: 2 Likes: 0 Date: October 13 2018
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 1030
Name: Sandy Williams
Year of Birth: 1963
Place of Birth: Namibia
Member Since: December 5 2010
Genres: Science Fiction
Books:
id: 24
Name: Melissa Landers
Year of Birth: 1951
Place of Birth: Croatia
Member Since: October 14 2012
Genres: Science Fiction, Young Adult
Books:
0.006035
id: 32
Title: Blackout
Genre: Horror
Author: Mira Grant
***
id: 232
Title: Crocodile on the Sandbank
Genre: Mystery
Author: Elizabeth Peters
***
id: 370
Title: Eternal Eden
Genre: Fantasy
Author: Nicole  Williams
***
id: 127
Title: Legend
Genre: Young Adult
Author: Marie Lu
***
id: 219
Title: Strands of Bronze and Gold
Genre: Young Adult
Author: Jane Nickerson
***
id: 65
Title: The 5th Wave
Genre: Young Adult
Author: Rick Yancey
***
id: 22
Title: The Children of Men
Genre: Fiction
Author: P.D. James
***
id: 327
Title: The Neverending Story
Genre: Fantasy
Author: Michael Ende
***
id: 42
Title: Tigers and Devils
Genre: Romance
Author: Sean Kennedy
***
id: 46
Title: Wicked as They Come
Genre: Science Fiction
Author: Delilah S. Dawson
***
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
id: 1594
Name: Steven Stickler
Year of Birth: 1978
Place of Birth: Angola
Member Since: February 7 2012
Genres: Children S Books
Books:
id: 312 Title: The Absolutely Amazing Adventures of Agent Auggie Spinoza
id: 1278
Name: Clarissa Goenawan
Year of Birth: 1962
Place of Birth: Sweden
Member Since: May 20 2017
Genres: Mystery, Science Fiction
Books:
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
